vignettes/extending.Rmd

```diff
@@ -316,7 +316,7 @@ test_that("autotest", {
   learner = LearnerRegrRpartSimple$new()
 
   # basic learner properties
-  expect_learner(learner, check_man = FALSE)
```

Comment:
I think this must be kept as is as otherwise the check fails (if the learner is not installed).
When adding a learner in a package it should be `TRUE`, maybe add this as a sentence if it is not present already